src/main/java/org/kainos/ea/auth/JwtGenerator.java

```diff
@@ -0,0 +1,49 @@
+package org.kainos.ea.auth;
+
+import java.security.KeyPair;
+import java.security.KeyPairGenerator;
+import java.security.NoSuchAlgorithmException;
+import java.security.interfaces.RSAPrivateKey;
+import java.security.interfaces.RSAPublicKey;
+import java.time.Instant;
+import java.util.Date;
+import java.util.UUID;
+
+import com.auth0.jwt.JWT;
+import com.auth0.jwt.JWTCreator.Builder;
+import com.auth0.jwt.algorithms.Algorithm;
+
+public class JwtGenerator {
+
+    public static final KeyPair KEYPAIR;
+
+    static {
+        try {
+            KEYPAIR = KeyPairGenerator.getInstance("RSA").generateKeyPair();
+        } catch (NoSuchAlgorithmException e) {
+            throw new RuntimeException(e);
```

Comment:
shouldn't throw a runtime exception, can catch NoSuchAlgorithmException and return a 500 from controller

Reply:
Fixed this